pyslackersweb/contexts.py

```diff
@@ -30,6 +30,5 @@
 
 
 async def postgresql_pool(app: web.Application) -> AsyncGenerator[None, None]:
-    async with asyncpg.create_pool(dsn=app["POSTGRESQL_DSN"]) as pool:
-        app["db"] = app["website_app"]["db"] = pool
-        yield
+    await pg.init(dsn=app["POSTGRESQL_DSN"])
```

Comment:
This is my biggest gripe about asyncpgsa, If we can discourage the global usage I’d prefer to explicitly set access on the app object(s).

Reply:
We can go the explicit way. `asyncpgsa` seems to favor the `singleton` but it supports both